armi/reactor/flags.py

```diff
@@ -181,6 +181,10 @@ def _toString(cls, typeSpec):
     """
     Make flag from string and fail if any unknown words are encountered.
 
+    .. impl:: Convert a flag to string
+        :id: I_ARMI_FLAG_TO_STR1
+        :implements: R_ARMI_FLAG_TO_STR
```

Comment:
We should delete this.  You have this req on the `def toString()` method below, that's the correct one.

The REASON we can't have this here is that we can't put requirement stubs on private methods.  (Any method/class/function that starts with a `_` is "private" and won't appear in our API docs.)

Sorry, does that make sense?